<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KirjausWg</class>
 <widget class="QWidget" name="KirjausWg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tositteen &amp;pvm</string>
       </property>
       <property name="buddy">
        <cstring>tositePvmEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="tositePvmEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Otsikko</string>
       </property>
       <property name="buddy">
        <cstring>otsikkoEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="otsikkoEdit"/>
     </item>
     <item>
      <widget class="QComboBox" name="tositetyyppiCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="viennit">
      <attribute name="title">
       <string>&amp;Viennit</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="viennitView">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="summaLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="lisaaRiviNappi">
           <property name="text">
            <string>Li&amp;sää rivi</string>
           </property>
           <property name="icon">
            <iconset resource="../pic/pic.qrc">
             <normaloff>:/pic/lisaarivi.png</normaloff>:/pic/lisaarivi.png</iconset>
           </property>
           <property name="shortcut">
            <string>F11</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="poistariviNappi">
           <property name="text">
            <string>Poista &amp;rivi</string>
           </property>
           <property name="icon">
            <iconset resource="../pic/pic.qrc">
             <normaloff>:/pic/poistarivi.png</normaloff>:/pic/poistarivi.png</iconset>
           </property>
           <property name="shortcut">
            <string>Del</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="kommentit">
      <attribute name="title">
       <string>&amp;Kommentit</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="kommentitEdit">
         <property name="placeholderText">
          <string>Kirjoita tähän tositteeseen liittyvät kommentit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="liitteet">
      <attribute name="title">
       <string>L&amp;iitteet</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QListView" name="liiteView">
         <property name="iconSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
         <property name="viewMode">
          <enum>QListView::IconMode</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="avaaNappi">
           <property name="text">
            <string>Avaa</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="lisaaliiteNappi">
           <property name="text">
            <string>Li&amp;sää</string>
           </property>
           <property name="icon">
            <iconset resource="../pic/pic.qrc">
             <normaloff>:/pic/lisaa.png</normaloff>:/pic/lisaa.png</iconset>
           </property>
           <property name="shortcut">
            <string>F11</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="poistaLiiteNappi">
           <property name="text">
            <string>Poista</string>
           </property>
           <property name="icon">
            <iconset resource="../pic/pic.qrc">
             <normaloff>:/pic/roskis.png</normaloff>:/pic/roskis.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tiliote">
      <attribute name="title">
       <string>&amp;Tiliote</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="tilioteBox">
         <property name="text">
          <string>Tosite on tiliote</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Tili</string>
         </property>
         <property name="buddy">
          <cstring>tiliotetiliCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="tiliotetiliCombo"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Ajalta</string>
         </property>
         <property name="buddy">
          <cstring>tiliotealkaenEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QDateEdit" name="tiliotealkaenEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>-</string>
           </property>
           <property name="buddy">
            <cstring>tilioteloppuenEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="tilioteloppuenEdit"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>732</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Tosite&amp;numero</string>
       </property>
       <property name="buddy">
        <cstring>tunnisteEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tyyppiLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tunnisteEdit">
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lukkoLabel">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../pic/pic.qrc">:/pic/lukittu.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="varoMerkki">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../pic/pic.qrc">:/pic/varoitus.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="apuriNappi">
       <property name="text">
        <string>A&amp;vustaja </string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/taikuri.png</normaloff>:/pic/taikuri.png</iconset>
       </property>
       <property name="shortcut">
        <string>F9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="laskuNappi">
       <property name="text">
        <string>&amp;Lasku</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/lasku.png</normaloff>:/pic/lasku.png</iconset>
       </property>
       <property name="shortcut">
        <string>F10</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="tallennaButton">
       <property name="text">
        <string>T&amp;allenna</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../pic/ok.png</normaloff>../pic/ok.png</iconset>
       </property>
       <property name="shortcut">
        <string>F12</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hylkaaNappi">
       <property name="text">
        <string>H&amp;ylkää</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/peru.png</normaloff>:/pic/peru.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="poistaNappi">
       <property name="text">
        <string>Poista</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/roskis.png</normaloff>:/pic/roskis.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
